client-v2/src/bundles/capiFeedBundle.ts

```diff
@@ -5,17 +5,18 @@ import { previewCapi, liveCapi } from 'services/frontsCapi';
 import { checkIsContent } from 'services/capiQuery';
 import { getPrefills } from 'services/editionsApi';
 import { Dispatch } from 'redux';
+import { State } from 'types/State';
+import { createSelectIsArticleStale } from 'shared/util/externalArticle';
 
-type FeedState = CapiArticle[];
+type FeedState = CapiArticle;
 
 const {
   actions: liveActions,
   reducer: capiLiveFeed,
   selectors: liveSelectors
 } = createAsyncResourceBundle<FeedState>('capiLiveFeed', {
-  indexById: false,
   selectLocalState: state => state.feed.capiLiveFeed,
-  initialData: []
+  indexById: true
```

Comment:
Instead of storing our feed articles in an array, we store them in an object and keep the incoming order elsewhere. This lets us update articles individually when we poll for changes, rather than having to update all of them at once.